### PR TITLE
docs: adding support for partial service ids

### PR DESCRIPTION
**Pull request by callmehiphop**
This adds support for full & partial service ID matching when deciding on whether or not to expand/collapse a navigation item.

### CODE REVIEW

**Comment by dwsupplee** [comment]
Just tested this out and it works great for the PHP docs, thanks! :)

**Comment by callmehiphop** [comment]
@dwsupplee glad to hear it :)

@daspecster did you have a chance to test this out?

**Comment by callmehiphop** [comment]
Nevermind, totally doesn't work for Python

**Comment by daspecster** [comment]
Seems to break it again :(

screen shot 2016-08-23 at 12 09 03 pm

File: site/src/app/docs/docs.controller.js [hunk 1]
```diff
@@ -38,7 +38,14 @@
     }
 
     function isActive(serviceId) {
-      return !!($state.params.serviceId || '').match(serviceId);
+      var serviceIdParam = $state.params.serviceId || '';
+
+      if (!manifest.matchPartialServiceId) {
```
Comment (callmehiphop) [inline site/src/app/docs/docs.controller.js]: @dwsupplee I had to take a peak at your JSON to remember why we needed to split the params and everything came back to me. I feel that making this configurable is the least destructive path. WDYT?

Comment (dwsupplee) [inline site/src/app/docs/docs.controller.js]: LGTM

Comment (daspecster) [inline site/src/app/docs/docs.controller.js]: Works for python!

**Pull request merged**